derive/src/trait_bounds.rs

```diff
@@ -143,23 +164,17 @@ pub fn add(
 
 		codec_types
 			.into_iter()
-			.for_each(|ty| {
-				where_clause.predicates.push(parse_quote!(#ty : #codec_bound))
-			});
+			.for_each(|ty| where_clause.predicates.push(parse_quote!(#ty : #codec_bound)));
 
 		let has_compact_bound: syn::Path = parse_quote!(#crate_path::HasCompact);
 		compact_types
 			.into_iter()
-			.for_each(|ty| {
-				where_clause.predicates.push(parse_quote!(#ty : #has_compact_bound))
-			});
+			.for_each(|ty| where_clause.predicates.push(parse_quote!(#ty : #has_compact_bound)));
 
-		skip_types
-			.into_iter()
-			.for_each(|ty| {
-				let codec_skip_bound = codec_skip_bound.as_ref().unwrap();
-				where_clause.predicates.push(parse_quote!(#ty : #codec_skip_bound))
-			});
+		skip_types.into_iter().for_each(|ty| {
+			let codec_skip_bound = codec_skip_bound.as_ref().unwrap();
```

Comment:
```suggestion
			let codec_skip_bound = codec_skip_bound.as_ref();
```

I know, not your code, but I think we don't need to unwrap here :D